improver_tests/blending/weighted_blend/test_PercentileBlendingAggregator.py

```diff
@@ -128,35 +144,35 @@
 BLENDED_PERCENTILE_DATA = np.array(
     [
         [[12.968588, 12.984587], [12.560181, 12.503394]],
-        [[12.990671, 12.984587], [14.356173, 12.503394]],
-        [[14.164387, 13.835985], [14.607758, 12.66411]],
-        [[14.855347, 14.404217], [14.736798, 13.913844]],
-        [[16.250134, 15.728171], [16.480879, 15.219085]],
-        [[17.458706, 17.408989], [17.481287, 17.0385]],
+        [[14.513496, 13.983413], [14.521752, 13.843809]],
+        [[15.451672, 15.030349], [14.965419, 14.030121]],
+        [[16.899292, 15.662327], [15.876251, 15.6937685]],
+        [[17.333557, 16.205572], [16.516674, 16.478855]],
+        [[17.458706, 17.408989], [17.481285, 17.0385]],
```

Comment:
The data in PERCENTILE DATA is reshaped and put into a cube that has 4 axes: one of those is "percentile".  For data to be sensible, it must be monotonically increasing along the "percentile" axis (ie you would always expect the 50th percentile of a dataset to be bigger than the 10th percentile, for example).

The code in the percentile aggregator does a linear interpolation between adjacent percentile values to find a probability associated with a value, and averages them across models.  But if the percentiles you've given it aren't monotonically ascending, the probabilities it finds in various intervals between "percentiles" won't be sensible.  By changing the percentile values to be in monotonically ascending order, you change the adjacent percentile values, change the interpolated value and therefore change the outputs.  The 0% and 100% values don't change because they're the only ones that aren't interpolated, but are taken straight from the max and min of the data.

Does that help?